src/clj/kefirnadar/application/handlers.clj

```diff
@@ -11,7 +11,7 @@
   "This handler returns all the users."
   [input]
   (println (:path-params input))
-  (let [users (q/users (client/db) (:path-params input))]
+  (let [users (q/ads (client/db) (:path-params input))]
```

Comment:
zašto u var pod nazivom `users` smeštamo rezultate izvršavanja upita pod nazivom `ads`?

Reply:
Ovo je staro, vec promenjeno xD